test/test_helper.rb

```diff
@@ -6,10 +6,13 @@
 require "webmock/minitest"
 
 Dir.glob(Rails.root.join("lib/*.rb")).each { |f| require_relative f }
+Dir.glob(Rails.root.join("test/factories/*.rb")).each { |f| require_relative f }
 
 WebMock.disable!
 
 class ActiveSupport::TestCase
+  include FactoryBot::Syntax::Methods
+
```

Comment:
You're not actually using this. Including this module allows you to remove `FactoryBot.` in your tests, so that `FactoryBot.build(:ruby_object, c: Integer)` becomes `build(:ruby_object, c: Integer)`. Since I like consistency, I feel like you should either remove the include, or remove `FactoryBot.` in the tests. Otherwise it feels like it leaves the door open for someone else to omit the `FactoryBot.` in other tests even though you're including it in these tests.

Reply:
I agree, lets keep things consistent.